CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.1] - 2023-04-11
```

Comment:
One note here: Following @wbaldoumas and @awesor 's discussion [here](https://github.com/wayfair-incubator/AeroSharp/pull/163#discussion_r1149621293) I'd expect this to be a minor version upgrade, not a patch.

Additionally, we need to actually change the version number in the csproj: https://github.com/wayfair-incubator/AeroSharp/blob/main/src/AeroSharp/AeroSharp.csproj#L6

I missed this in my initial review, sorry.

Reply:
got it, let me fix it